test/integration/krane_deploy_test.rb

```diff
@@ -1826,6 +1826,14 @@ def test_succeeds_with_deploy_method_override
     assert_deploy_success(deploy_fixtures("deploy-method-override"))
   end
 
+  def test_duplicate_kind_resource_definition
+    result = deploy_global_fixtures("crd", subset: ["deployment.yml"])
+    assert_deploy_success(result)
+
+    result = deploy_fixtures("crd", subset: ["web.yml"], global_timeout: 30)
+    assert_deploy_success(result)
+  end
+
```

Comment:
This test is passing just fine on master branch, I don't think it's forcing the failure you think it is

Reply:
Should be fixed now, the testing code applies a prefix to the Kind so in that case there is no duplicate.